host-configs/apple/macOS_base.cmake

```diff
@@ -6,24 +6,38 @@ message("CONFIG_NAME = ${CONFIG_NAME}")
 
 set(CMAKE_C_COMPILER "/usr/bin/clang" CACHE PATH "")
 set(CMAKE_CXX_COMPILER "/usr/bin/clang++" CACHE PATH "")
+set(CMAKE_FORTRAN_COMPILER "/usr/bin/gfortra`" CACHE PATH "")
 set(ENABLE_FORTRAN OFF CACHE BOOL "" FORCE)
 
 set(ENABLE_MPI ON CACHE PATH "")
-set(MPI_C_COMPILER "${HOMEBREW_DIR}/bin/mpicc" CACHE PATH "")
-set(MPI_CXX_COMPILER "${HOMEBREW_DIR}/bin/mpicxx" CACHE PATH "")
-set(MPIEXEC "${HOMEBREW_DIR}/bin/mpirun" CACHE PATH "")
+#set(MPI_C_COMPILER "${HOMEBREW_DIR}/bin/mpicc" CACHE PATH "")
+#set(MPI_CXX_COMPILER "${HOMEBREW_DIR}/bin/mpicxx" CACHE PATH "")
+#set(MPI_Fortran_COMPILER "${HOMEBREW_DIR}/bin/mpifort" CACHE PATH "")
+#set(MPIEXEC "${HOMEBREW_DIR}/bin/mpirun" CACHE PATH "")
+
+set(MPI_C_COMPILER "mpicc" CACHE PATH "")
+set(MPI_CXX_COMPILER "mpicxx" CACHE PATH "")
+set(MPI_Fortran_COMPILER "mpifort" CACHE PATH "")
+set(MPIEXEC "mpirun" CACHE PATH "")
```

Comment:
can you revert these changes? you should be able to just set a different homebrew path if you are not using the default one.